apps/chat/src/store/conversations/conversations.epics.ts

```diff
@@ -2308,12 +2293,12 @@ const uploadConversationsByIdsEpic: AppEpic = (action$, state$) =>
 
 const saveConversationEpic: AppEpic = (action$) =>
   action$.pipe(
-    filter(
-      (action) =>
-        ConversationsActions.saveConversation.match(action) &&
-        !action.payload.isMessageStreaming, // shouldn't save during streaming
-    ),
+    filter((action) => ConversationsActions.saveConversation.match(action)),
```

Comment:
```suggestion
    filter(ConversationsActions.saveConversation.match),
```

lets fix payload type